src/pages/[...slug].tsx

```diff
@@ -0,0 +1,352 @@
+};
+
+const ContentPage: NextPage<ContentPiece> = ({
+    authors,
+    category,
+    image,
+    title,
+    description,
+    contentDate,
+    tags,
+    slug,
+}) => {
+    const [ratingStars, setRatingStars] = useState(0);
+    const [modalStage, setModalStage] = useState<modalStages>('closed');
+
+    const relatedContent = getRelatedContent(slug);
+    const series = getSeries(slug);
+    const slugList = slug.split('/');
+    const tertiaryNavItems = getTertiaryNavItems(slugList[slugList.length - 2]);
+
+    const vidOrPod = category === 'Video' || category === 'Podcast';
+
+    const ratingSection = (
+        <div
+            sx={{
+                display: 'flex',
+                justifyContent: 'end',
+                alignItems: 'center',
+            }}
+        >
+            <span>Rate this {category.toLowerCase()}</span>
+            <ContentRating
+                stars={ratingStars}
+                onRate={i => {
+                    setRatingStars(i); // This should update both rating sections but it doesn't.
+                    setModalStage(i === 3 ? 'text' : 'checkbox');
+                }}
+            />
+        </div>
+    );
+
+    const contentHeader = (
+        <>
+            <Eyebrow sx={{ marginBottom: 'inc30' }}>{category}</Eyebrow>
+            <TypographyScale
+                variant="heading2"
+                sx={{
+                    marginBottom: ['inc20', null, null, 'inc30'],
+                }}
+            >
+                {title}
+            </TypographyScale>
+            {!vidOrPod ? (
+                <div sx={socialFlexContainerStyles}>
+                    <SpeakerLockup
+                        name={authors?.join(',')}
+                        title={`${contentDate}`}
+                    />
+                    {SocialButtons}
+                </div>
+            ) : (
+                <TypographyScale
+                    variant="body3"
+                    color="secondary"
+                    customStyles={{
+                        display: 'block',
+                        marginBottom: 'inc30',
+                    }}
+                >
+                    {contentDate}
+                </TypographyScale>
+            )}
+            {tags && (
+                <div sx={socialFlexContainerStyles}>
+                    <TagSection tags={tags} />
+                    {vidOrPod && SocialButtons}
+                </div>
+            )}
+            {image && (
+                <div sx={imageStyles}>
+                    <Image
+                        alt={image.alt}
+                        src={image.url}
+                        sx={{
+                            borderRadius: 'inc30',
+                            objectFit: 'cover',
+                        }}
+                        layout="fill"
+                    />
+                </div>
+            )}
+            {!vidOrPod && ratingSection}
+        </>
+    );
+
+    const contentFooter = (
```

Comment:
Did we mean to keep this here?

Reply:
Yeah it's a work in progress since the footer will also go on this page.